run_tests.sh

```diff
@@ -1,7 +0,0 @@
-#!/usr/bin/env bash
-
-export PASH_TOP=${PASH_TOP:-$(git rev-parse --show-toplevel --show-superproject-working-tree)}
-
-( echo "Running intro tests..."; cd "$PASH_TOP/evaluation/intro"; ./test.sh ) &&
-( echo "Running interface tests..."; cd "$PASH_TOP/evaluation/tests/interface_tests"; ./run.sh ) &&
```

Comment:
@nvasilakis I wanted these to be composed with `&&` so that tests stop whenever one of the "increasingly complex suites" errs. Is there a reason why you removed those?